bench/workgen/validate_mirror_tables.py

```diff
@@ -64,7 +64,12 @@ def get_mirrors(db_dir, db_files):
             continue
         db_files_remaining.remove(filename)
         mirror_filename = get_mirror_file(db_dir, f'table:{filename}')
-        if mirror_filename is not None:
+
+        # At this point, there is no guarantee that the database contains all the base/mirror pairs.
+        # It is possible to have a base and no associated mirror and vice-versa. This may happen
+        # when a drop was occurring when a snapshot of the database was taken and fed into this
```

Comment:
NP: both instances of "was" should be "is".

Reply:
Updated in 7387c02105ead5b36ce7d4e9a13e01b97a3b21cd